res/style.css

```diff
@@ -17,9 +17,6 @@ body, #root, .profileViewer {
   display: flex;
   flex-flow: column nowrap;
   flex: 1;
-  -webkit-user-select: none;
-  -moz-user-select: none;
-  user-select: none;
```

Comment:
Since this doesn't actually help, let's revert this change. Also, Firefox doesn't support unprefixed `user-select` yet, it needs `-moz-user-select`.